smarts/core/vehicle.py

```diff
@@ -374,13 +375,23 @@ def attach_sensors_to_vehicle(sim, vehicle, agent_interface, mission_planner):
             )
 
         if agent_interface.waypoints:
-            vehicle.attach_waypoints_sensor(
-                WaypointsSensor(
-                    vehicle=vehicle,
-                    mission_planner=mission_planner,
-                    lookahead=agent_interface.waypoints.lookahead,
+            # TODO
```

Comment:
@davidrusu @JVillella @Gamenot This is the design where I couldn't make the call. Currently, I replace the original waypoints with a u-turn trajectory, which makes it easier for agents to do the task, but the downsides are: 1. mix the waypoints and desired trajectory conceptually 2. The changes might not be expected to users.

Reply:
Let's talk about it in our team sync this evening?